config/env_config.go

```diff
@@ -482,6 +494,34 @@ func (c EnvConfig) getCustomCABundle(context.Context) (io.Reader, bool, error) {
 	return bytes.NewReader(b), true, nil
 }
 
+// GetIgnoreConfiguredEndpoints is used in knowing when to disable configured
+// endpoints feature.
+func (c EnvConfig) GetIgnoreConfiguredEndpoints(context.Context) (bool, bool, error) {
```

Comment:
Why `context.Context`? Don't see it used anywhere, it's also not possible for the caller to propagate it with the structure of how these are used (I see you're setting it to `context.Background()` later on).

Reply:
Oh I see it's part of that static `resolver` signature. I would just not propagate it further down wherever it's not needed.

Reply:
sure, but in this impl in env config and shared config, i dont propagate it down to any further functions. feel free to elaborate if im misunderstanding